jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/JKubeServiceHubTest.java

```diff
@@ -59,10 +61,15 @@
   private BuildServiceConfig buildServiceConfig;
 
   private JKubeServiceHub.JKubeServiceHubBuilder commonInit() {
+    return commonInit(true);
+  }
+
+  private JKubeServiceHub.JKubeServiceHubBuilder commonInit(boolean connectToCluster) {
```

Comment:
Follow the existing approach and take advantage of the builder.

If you were to add parameters for every override, then tests would be less declarative, fluent, and readable.

e.g. 
In line 118 you are overrding the cluster access **and** the platform mode.
---> `commonInit().platformMode(RuntimeMode.KUBERNETES).offline(true)`